mlir/CMakeLists.txt

```
@@ -12,9 +12,8 @@ set(MQT_MLIR_MIN_VERSION 19.0)
 
 # MLIR must be installed on the system
 find_package(MLIR REQUIRED CONFIG)
-if(MLIR_VERSION VERSION_LESS MQT_MLIR_MIN_VERSION)
-  message(FATAL_ERROR "MLIR version must be at least ${MQT_MLIR_MIN_VERSION}")
-endif()
+# if(MLIR_VERSION VERSION_LESS MQT_MLIR_MIN_VERSION) message(FATAL_ERROR "MLIR version must be at
+# least ${MQT_MLIR_MIN_VERSION} but found ${MLIR_VERSION}") endif()
```

Why was this commented out?
```suggestion
if(MLIR_VERSION VERSION_LESS MQT_MLIR_MIN_VERSION)
  message(FATAL_ERROR "MLIR version must be at least ${MQT_MLIR_MIN_VERSION}")
endif()
```